app/frontend/src/assets/scss/style.scss

```diff
@@ -0,0 +1,35 @@
+/* sticky footer */
+html,
+body {
+  height: 100%;
+}
+body {
+  display: flex;
+  flex-direction: column;
+}
+body > .v-content {
+  flex: 1 0 auto;
+}
+
+.BC-Gov-PrimaryButton.v-btn {
+  background-color: #003366;
+  border: none;
+  border-radius: 4px;
+  color: white !important;
```

Comment:
Would be best to stay away from !important as much as possible (though Vuetify can make that tricky).
We can re-assess when we make some theming decisions

Reply:
ok